### PR TITLE
Inventory ItemStats Toggle Option While Viewing Banking

### PR DESCRIPTION
**Pull request by basicDamen**
Adds the option to enable/disable the plugin while banking.

Clarification: This only adds the option to disable the INVENTORY side WHILE banking. If enabled, you can walk around and have item stats, though, the moment you open a bank, the inventory-side of item stats is disabled.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatConfig.java [hunk 1]
```diff
@@ -69,6 +69,17 @@ default boolean theoretical()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "bankToggle",
+			name = "Show While Banking",
+			description = "Show inventory tooltip while viewing bank"
+	)
```
Comment (SoyChai) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatConfig.java]: Reduce the tabbing here so it lines up with the other ConfigItems.

**Comment by Nightfirecat** [comment]
Needs rebase

**Comment by xDemoN** [comment]
Any update on this? I can rebase it

**Comment by Nightfirecat** [comment]
This was added in #10258

**Pull request closed without merging**